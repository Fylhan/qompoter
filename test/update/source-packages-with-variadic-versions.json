{
	"name": "install-ok/qompoter-versionstar-offline",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"cp/fylhan": "v1.*",
		"cp/acme-cp": "v1.*-lib",
		"fylhan/posibrain": "v0.*"
	},
	"repositories": {
		"fylhan/posibrain": "https://github.com"
	}
}
