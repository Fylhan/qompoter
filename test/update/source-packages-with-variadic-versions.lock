{
  "last": "1147fc622a5cb562f7ca02450576431a",
  "name": "install-ok/qompoter-versionstar-offline",
  "date": "2021-02-24T15:46:28+01:00",
  "require": {
    "cp/fylhan": { "version": "v1.1.10", "type": "qompotist-fs", "url": "qompoter-repo/cp/fylhan/v1.1.10", "md5sum": "9d57702586a06dcd5de4d717d936c9d6" }
    ,"cp/acme-cp": { "version": "v1.0-lib", "type": "qompotist-fs", "url": "qompoter-repo/cp/acme-cp/v1.0-lib", "md5sum": "115f3cd6f4a5dec31fb31b9cf39fa855" }
    ,"fylhan/posibrain": { "version": "v0.2", "type": "git", "url": "https://github.com/fylhan/posibrain.git", "md5sum": "1147fc622a5cb562f7ca02450576431a" }
  }
}
