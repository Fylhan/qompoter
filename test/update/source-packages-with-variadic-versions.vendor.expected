vendor
vendor/acme-cp
vendor/acme-cp/qompoter.json
vendor/acme-cp/qompoter.pri
vendor/acme-git/qompoter.pri
vendor/fylhan
vendor/fylhan/qompoter.json
vendor/fylhan/qompoter.pri
vendor/lib_linux_64
vendor/lib_linux_64/acme-cp_fake.a
vendor/qompote.pri
vendor/vendor.pri
